packages/traildb/traildb.0.1/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "Didier Wenzek <didier.wenzek@acidalie.com>"
+authors: "Didier Wenzek <didier.wenzek@acidalie.com>"
+homepage: "https://github.com/didier-wenzek/ocaml-traildb"
+bug-reports: "https://github.com/didier-wenzek/ocaml-traildb/issues"
+license: "MIT"
+dev-repo: "https://github.com/didier-wenzek/ocaml-traildb.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: ["jbuilder" "runtest" "-j" jobs]
+depends: [
+  "uuidm"
+  "jbuilder" {build}
+  "oUnit" {test}
+]
+depexts: [
+  ["debian"]
+  ["libtraildb-dev"]
```

Comment:
This should be written ```[["debian"] ["libtraildb-dev"]]``` to work.
Could you also look at other distributions ? Links to look for the necessary package(s) here: https://github.com/ocaml/opam-depext/blob/master/README.md

Reply:
Indeed, the syntax was incorrect. I also added two distributions.